src/main/java/reposense/dataobject/RepoConfiguration.java

```diff
@@ -15,9 +15,8 @@
     private String repoName;
     private String branch;
     private String displayName;
-    private Date fromDate;
-    private Date toDate;
-
+    private Date sinceDate;
+    private Date untilDate;
     private transient boolean needCheckStyle = false;
```

Comment:
should have a newline to separate `private transient` from `private`, and `private` variables grouped together